samples/snippets/csharp/VS_Snippets_CLR_System/system.IO.Pipes.NamedPipeClientStream_ImpersonationSample1/cs/NamedPipeClientStream_ImpersonationSample.csproj

```diff
@@ -0,0 +1,62 @@
+﻿<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Can this csproj be an SDK-style?

Reply:
Generated it from VS 2019.
Which style should I apply?

Reply:
Similar to:
https://github.com/dotnet/docs/blob/ac15aa1802911baa07f006ddb2bee06d864b164d/docs/csharp/programming-guide/main-and-command-args/snippets/top-level-statements-5/top-level-statements.csproj

Reply:
Thanks, I'll do that